wuzz.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: wuzz
+  version: 0.5.0
+  epoch: 0
+  description: Interactive cli tool for HTTP inspection
+  copyright:
+    - license: AGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/asciimoo/wuzz
+      tag: v${{package.version}}
+      expected-commit: 0935ebdf55d223abbf0fd10cb8f0f9c0a323ccb7
+
+  - uses: go/bump
+    with:
+      deps: github.com/tidwall/gjson@v1.9.3 golang.org/x/net@v0.23.0 golang.org/x/sys@v0.21.0
+
+  - uses: go/build
+    with:
+      output: wuzz
+      packages: .
+
+  - uses: strip
+
+update:
+  enabled: true
```

Comment:
Can you please add a basic test  , [example here  ](https://github.com/wolfi-dev/os/blob/main/age.yaml#L32-L40)